R/install-bitbucket.r

```diff
@@ -56,20 +62,20 @@ remote_download.bitbucket_remote <- function(x, quiet = FALSE) {
   if (!quiet) {
     message("Downloading bitbucket repo ", x$username, "/", x$repo, "@", x$ref)
   }
-
   dest <- tempfile(fileext = paste0(".zip"))
   src <- paste("https://bitbucket.org/", x$username, "/", tolower(x$repo), "/get/",
     x$ref, ".zip", sep = "")
-
-  if (!is.null(x$password)) {
+  if (!is.null(x$auth_token)) {
+    userpwd <- strsplit(x$auth_token, ":")[[1]]
+    stopifnot(!is.na(userpwd[2])) # stop if no colon in auth_token
```

Comment:
I think this test should be `length(userpwd) == 2` and we should have a more useful error message

```r
parts <- strsplit(x$auth_token, ":")[[1]]
if (length(parts) != 2) {
  stop ("`auth_token` needs a username and password separated by a colon", call. = FALSE)
}
```